dissect/target/plugins/os/unix/linux/fortios/_os.py

```diff
@@ -94,20 +155,101 @@ def ips(self) -> list[str]:
 
     @export(property=True)
     def version(self) -> str:
-        if config_fh := self._find_config_file():
-            with config_fh:
-                return config_fh.readline().split("=")[1]
-
+        """Return FortiOS version"""
+        if self._version:
+            return parse_version(self._version)
         return "FortiOS Unknown"
 
     @export(property=True)
     def os(self) -> str:
-        return OperatingSystem.FORTIGATE.value
+        """Return `fortios`"""
+        return OperatingSystem.FORTIOS.value
+
+    @export(record=FortiOSUserRecord)
+    def users(self) -> Iterator[FortiOSUserRecord]:
+        """Return local users of the FortiOS system"""
+
+        # Administrative users
+        for username, entry in self._config["global-config"]["system"]["admin"].items():
```

Comment:
No try/excepts needed here? All the keys in this function always exist?

Reply:
Added in [e765ee2](https://github.com/fox-it/dissect.target/pull/509/commits/e765ee29af0ff2bb8f524c6c5eb01eb7932f9ff6).